subsys/mgmt/hawkbit/Kconfig

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2020 Linumiz
+# SPDX -License-Identifier: Apache-2.0
+
+menuconfig HAWKBIT
+	bool "Hawkbit Firmware Over-the-Air support"
+	select NVS
```

Comment:
This is OK for the first iteration, but I think this could be improved by not imposing a particular storage mechanism on the user. For example, if the user is selecting littlefs and has already an instance of it, hawkbit should be able to use that instead of NVS.